File: tabFragment/src/main/java/com/tabfragment/FragmentFour.java
package com.tabfragment;

import android.os.Bundle;
import android.support.annotation.Nullable;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

/**
 * by y on 2016/10/5.
 */

public class FragmentFour extends BaseFragment {

    public static FragmentFour startFragment() {
        return new FragmentFour();
    }

    @Nullable
    @Override
    public View onCreateView(LayoutInflater inflater, @Nullable ViewGroup container, @Nullable Bundle savedInstanceState) {
        return getLayoutInflater(savedInstanceState).inflate(R.layout.fragment_four, null);
    }
}

File: fragment_tab/src/main/java/fragment/develop/android/tab/FragmentFour.kt
package fragment.develop.android.tab

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup

class FragmentFour : BaseFragment() {

    override fun onCreateView(
        inflater: LayoutInflater,
        container: ViewGroup?,
        savedInstanceState: Bundle?
    ): View? {
        return inflater.inflate(R.layout.fragment_four, container, false)
    }

    companion object {
        fun startFragment(): FragmentFour {
            return FragmentFour()
        }
    }

}
